packages/twenty-server/src/metadata/field-metadata/field-metadata.resolver.ts

```diff
@@ -35,4 +58,108 @@ export class FieldMetadataResolver {
       workspaceId,
     });
   }
+
+  @ResolveField(() => RelationFromOneSideDTO, { nullable: true })
+  async relationFromThisFieldPointOfView(
+    @Parent() fieldMetadata: FieldMetadataDTO,
+  ): Promise<RelationFromOneSideDTO | null> {
+    if (fieldMetadata.type !== FieldMetadataType.RELATION) {
```

Comment:
I think all this logic should be moved to a service so it can be re-used in other places in the BE

Reply:
We can put this into the `field-metadata.service` and then call it from there